src/commands/addons/admin/manifest/generate.ts

```diff
@@ -60,19 +59,19 @@ The file has been saved!`, ];
     const questions = [{
       type: 'input',
       name: 'id',
-      message: 'Enter slugname/manifest id:',
+      message: 'Enter slug:',
       default: flags.slug,
       validate: (input: any): boolean => {
         if (input.trim() === '' || !isNaN(input)) {
```

Comment:
@mathias I am confusing by this check, can we just let the create fail with bad slug format or something?  Maybe this could just be reduced to a `''` check with a message that we were expecting non-empty

Reply:
@ransombriggs I am not updating the logic here; I'll pair with you and/or @RasPhilCo over next 2 weeks on that sort of work. Just focused on the output messages here -- if that's OK, can you approve?